lektor/db.py

```diff
@@ -410,7 +410,7 @@ def get_record_label_i18n(self):
                 label = self.get_fallback_record_label(lang)
             rv[lang] = label
         # Fill in english if missing
-        if not rv:
+        if "en" not in rv:
```

Comment:
Wouldn't this add an `en` language, even for sites that don't have an `en` translation?